static/js/muting_ui.js

```diff
@@ -158,6 +163,59 @@ export function toggle_topic_mute(message) {
     }
 }
 
+export function persist_user_mute(user_id) {
+    last_user_update = timestamp_ms();
+    channel.post({
+        url: "/json/users/me/muted_users/" + user_id,
+        idempotent: true,
+    });
+}
```

Comment:
Structurally, it would be better for this commit to be after the commits that implement the actual hiding behavior in this PR.  While this commit is mostly complete, we won't want to merge a commit claiming that muting users does something, when it doesn't yet.